packages/transaction-controller/src/TransactionController.ts

```diff
@@ -898,6 +902,50 @@ export class TransactionController extends BaseController<
     this.incomingTransactionHelper.stop();
   }
 
+  /**
+   * Adds external provided transaction to state as confirmed transaction.
+   *
+   * @param transactionMeta - TransactionMeta to add transactions.
+   * @param transactionReceipt - TransactionReceipt of the external transaction.
+   * @param baseFeePerGas - Base fee per gas of the external transaction.
+   */
+  async confirmExternalTransaction(
+    transactionMeta: TransactionMeta,
+    transactionReceipt: TransactionReceipt,
+    baseFeePerGas: Hex,
+  ) {
+    // Create logger
+    const log = createModuleLogger(projectLogger, 'confirmExternalTransaction');
```

Comment:
This is a cool feature but I understood these require enabling explicitly via environment variables, and are more beneficial when adding verbose logging of complex processes.

Given our case is a simple error we likely always to show, as in the extension, would it be best to use a standard `console.error` as we do elsewhere?

Reply:
Removed logger in https://github.com/MetaMask/core/pull/1625/commits/916e06ad7b85ee5e488f1e0b6645c7534d879519